meson.build

```diff
@@ -176,15 +176,46 @@ subdir('milter/client')
 subdir('milter/server')
 subdir('milter/manager')
 subdir('libmilter')
-libruby = dependency('ruby-3.2', 'ruby-3.1', 'ruby-3.0', 'ruby-2.7', 'ruby',
-                     required: true)
+
+if meson.version().version_compare('>=0.60.0')
+  libruby = dependency('ruby-3.2', 'ruby-3.1', 'ruby-3.0', 'ruby-2.7', 'ruby',
+                       required: true)
+else
+  # depencency():
+  # Before 0.60.0 only a single dependency name was allowed.
+  # https://mesonbuild.com/Reference-manual_functions.html#dependency
+  foreach ruby: ['ruby-3.2', 'ruby-3.1', 'ruby-3.0', 'ruby-2.7', 'ruby']
+    libruby = dependency(ruby, required: false)
+    if libruby.found()
+      break
+    endif
+  endforeach
+  if not libruby.found()
+    error('Ruby not found.')
+  endif
+endif
+
 ruby_install_dir = get_option('ruby-install-dir')
-if ruby_install_dir == 'site'
-  ruby_install_dir = libruby.get_variable('sitelibdir')
-elif ruby_install_dir == 'vendor'
-  ruby_install_dir = libruby.get_variable('vendorlibdir')
+
+if meson.version().version_compare('>=0.58.0')
```

Comment:
We don't need this check because Ruby provides only `.pc`. We can always use `get_variable(pkgconfig: ...)`.

Reply:
It's also commonly used for consistency between "internal" dependencies provided by a subproject dependency fallback, but let's face it, that isn't very likely for ruby.

Reply:
Thanks for the advice!
I have fixed this!